### PR TITLE
Fixed document app to use Django forms

### LINKED ISSUE #150: Django/Bootstrap forms for information pages
Use Django forms for information pages forms (editor and attachment upload). Why didn't we use them in the first place?

### PR DESCRIPTION
**Pull request by invliD**
Fixes #150.

### CODE REVIEW

**Comment by Bartzi** [comment]
could you try to enlarge the forms? They are not wide enough:
image

File: _1327/documents/templates/attachments.html [hunk 1]
```diff
@@ -1,34 +1,20 @@
...
-		<div class="form-group {% if 'file' in form.errors %}has-error{% endif %}">
-			<label for="inputFile" class="col-sm-2 control-label">{% trans "Attachment" %}</label>
-			<div class="col-sm-9">
-				<input type="file" class="form-control" name="file" value="{{ form.data.file }}" placeholder="{% trans "Your Attachment..." %}">
-			</div>
-		</div>
-		<div class="form-group">
-			<div class="col-sm-offset-2 col-sm-4">
+		{% buttons %}
+			<div class="col-md-offset-3 col-md-4">
```
Comment (janno42) [inline _1327/documents/templates/attachments.html]: col-md-offset-2

**Comment by janno42** [comment]
... and the save button has a too high offset

**Comment by invliD** [comment]
Different versions of ``django-bootstrap3`` had different defaults. Should be fixed now.

**Comment by Bartzi** [comment]
:+1:

**Pull request merged**